data/reusables/package_registry/package-immutability.md

```diff
@@ -1,3 +1,5 @@
-To prevent confusion and build problems, {% data variables.product.prodname_dotcom %} permanently reserves a package's name and version number. Even if an entire package is deleted, you cannot reuse the deleted package name in any repository owned by the same account.
+**Public packages:** To prevent confusion and build problems, {% data variables.product.prodname_dotcom %} permanently reserves a package's name and version number. Even if an entire package is deleted, you cannot reuse the deleted package name in any repository owned by the same account.
+
+**Private packages:** If all versions of a private package are deleted, the user can re-create the package with any version number they want. When all versions of a package have been deleted, republishing over those deleted versions (i.e. publishing a package with the same name and version) should be the same as publishing a new package for the first time.
```

Comment:
What do you think of condensing the `Private packages` section into:

> **Private packages:** If all versions of a private package are deleted, you can publish a package with a previously used name and version.

Reply:
Much better. Thanks!